src/core/markdown.js

```diff
@@ -86,6 +86,14 @@ const inlineElems = new Set([
   "var",
 ]);
 
+const renderer = new marked.Renderer();
+renderer.code = function(code, language, isEscaped) {
+  if (language === "webidl") {
+    return `<pre class="idl">${code}</pre>`;
+  }
+  return marked.Renderer.prototype.code.call(this, code, language, isEscaped);
```

Comment:
🤮 ... I wonder if there is a better way of doing this? Wonder if something like this would work:

```JS
class IDLRender extends marked.Renderer {
   code(code, language, isEscaped) {
     super.code(code, language, isEscaped);
   }
}
```

I'm not sure...

Reply:
Worked :D